pkgs/development/libraries/grpc/default.nix

```diff
@@ -20,15 +20,15 @@
 
 stdenv.mkDerivation rec {
   pname = "grpc";
-  version = "1.44.0"; # N.B: if you change this, please update:
+  version = "1.45.2"; # N.B: if you change this, please update:
     # pythonPackages.grpcio-tools
     # pythonPackages.grpcio-status
```

Comment:
Both updated on python-updates #166489 and failing because they require grpc>=1.45.0, hence the reason for this update.